### PR TITLE
Add bootstrap input-group-addon with option fieldAddonLeft/Right

### PR DESCRIPTION
**Pull request by quietlyconfident**
Implements the second of the feature requests in #273

### CODE REVIEW

File: src/directives/decorators/bootstrap/default.html [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-         placeholder="{{form.placeholder}}"
-         class="form-control {{form.fieldHtmlClass}}"
-         id="{{form.key.slice(-1)[0]}}"
-         ng-model-options="form.ngModelOptions"
-         ng-model="$$value$$"
-         ng-disabled="form.readonly"
-         schema-validate="form"
-         name="{{form.key.slice(-1)[0]}}"
-         aria-describedby="{{form.key.slice(-1)[0] + 'Status'}}">
+  <div ng-class="{'input-group': (form.fieldAddonLeft || form.fieldAddonRight)}">
```
Comment (davidlgj) [inline src/directives/decorators/bootstrap/default.html]: I like your commit in general, but this introduces an extra div for everyone... I'm not sure its a good idea, but maybe we could try having two versions and switching between them with an ng-if?

Something like:
```
<div class="input-group" ng-if="(form.fieldAddonLeft || form.fieldAddonRight)}">
...
</div>
<input  ng-if="!(form.fieldAddonLeft || form.fieldAddonRight)}"  .... >
```
What do you think?

**Comment by quietlyconfident** [comment]
The bug in question is #164 .  To not have the doc, we could go bigger, and do ng-if on the outermost div, but that's not very DRY.

Also I took a look at the failed test cases, I think it is not failing because of an actual bug. But rather because the implementation of the test cases is very tightly coupled to the implementation of the HTML; putting in the extra div layer you noted, which should have no other effect, breaks those test cases.

**Comment by davidlgj** [comment]
Yeah I figured that the tests where breaking because of the extra div. I agree that the outermost div is not a good idea. 

The fear I have is mostly due to peoples CSS breaking, but with proper documentation/changelog that should be easily fixable. I have time this Friday to actually do some coding and not just answer issues, so I'll take a look at it then if that's OK with you?

**Comment by quietlyconfident** [comment]
Sounds great!  Thanks for considering the patch.

**Comment by davidlgj** [comment]
We'll time slipped away for me, I got a lot of schema form things done, but I haven't had time to merge this yet, but it will be done before next release.

**Comment by davidlgj** [comment]
Merged into development, I found the fix for #164 so I took the liberty of rearranging it so we got rid of the extra div. That fixed the test as well. I need to document it though before there can be a release. 

Thanks again!

**Pull request merged**